Octokit.Tests.Integration/Clients/AssigneesClientTests.cs

```diff
@@ -38,4 +41,28 @@ public AssigneesClientTests()
         var assignees = await _github.Issue.Assignee.GetAllForRepository(_context.RepositoryOwner, _context.RepositoryName);
         Assert.True(assignees.Any(u => u.Login == Helper.UserName));
     }
+
+    [IntegrationTest]
+    public async Task CanAddAndRemoveAssignees()
+    {
+        var newAssignees = new AssigneesUpdate(new List<string>() { _context.RepositoryOwner });
+        var newIssue = new NewIssue("a test issue") { Body = "A new unassigned issue" };
+
+        var issue = await _issuesClient.Create(_context.RepositoryOwner, _context.RepositoryName, newIssue);
+
+        Assert.NotNull(issue);
+
+        var addAssignees = await _github.Issue.Assignee.AddAssignees(_context.RepositoryOwner, _context.RepositoryName, issue.Id, newAssignees);
```

Comment:
You'll need to use `issue.Number` here and on the other actions below (the user-friendly id) rather than `issue.Id` (essentially the database column).